src/stories/story-utils/storeSetup.ts

```diff
@@ -3,23 +3,37 @@ import Vue from 'vue'
 import {namespace} from 'vuex-class'
 const flowVuexNamespace = namespace('flow')
 const builderVuexNamespace = namespace('builder')
-import {IBlock, IFlow} from '@floip/flow-runner'
+import {findFlowWith, IBlock, IContext, IFlow} from '@floip/flow-runner'
+import {get} from 'lodash'
 
-export const baseMounted = async function (BLOCK_TYPE: string, blockTypeStore: IRootState): Promise<any> {
+/**
+ * Safe register block module
+ * Because some weird race condition is leading to modules not getting unregistered when clicking between stories before the next story re-registers
+ */
+export const safeRegisterBlockModule = async function(BLOCK_TYPE: string, blockTypeStore: IRootState) {
   // @ts-ignore - TS2551: Property '$store' does not exist on type
   if (this.$store.state['flow'][BLOCK_TYPE]) {
-    //some weird race condition is leading to modules not getting unregistered when clicking between stories before the next story re-registers
     // @ts-ignore - TS2551: Property '$store' does not exist on type
     this.$store.unregisterModule(['flow', BLOCK_TYPE])
   }
   // todo: this will end up in `flow_addBlankBlockByType` once we get async import builds working
   // @ts-ignore - TS2551: Property '$store' does not exist on type
-  this.$store.registerModule(['flow', BLOCK_TYPE], blockTypeStore) 
+  this.$store.registerModule(['flow', BLOCK_TYPE], blockTypeStore)
+}
+
+export const baseMounted = async function (this: any, BLOCK_TYPE: string, blockTypeStore: IRootState): Promise<any> {
+  await safeRegisterBlockModule.bind(this)(BLOCK_TYPE, blockTypeStore)
+
+  const firstFlowId = get(this.$store.state, 'flow.firstFlowId')
+  let flow
+  if (firstFlowId) {
+    flow = findFlowWith(firstFlowId, get(this.$store.state, 'flow') as unknown as IContext)
```

Comment:
had to use the same flow if it already exists

Reply:
Can you explain a bit more about why this change to base mounted around finding firstFlowId is necessary? Wouldn't it be enough to just add the safeRegisterModel function?

Reply:
I found that in stories we want to have `non starting block`, we fake a case block as activated block. For that, we [use `flow_addBlankBlockByType` to add the case block](https://github.com/FLOIP/flow-builder/pull/12/files#diff-adcd1d3800c16e71c7dcad6f8fbce2afbaf7b6848fce9d877e6eb4241ce59c95R95), but this function doesn't add the case block in the same flow as the first block (non activated) but in the flow mentioned as `firstFlowId`. In fact, we cannot specify the destination `flowId` under `flow_addBlankBlockByType` for now